More on filtering features with form (#4001)
qgis/QGIS-Documentation: source/docs/user_manual/working_with_vector/attribute_table.rst

in :ref:`select_by_value`, where you can find descriptions of all operators
and selecting modes.

.. _figure_filter_select_form:

.. figure:: img/tableFilteredForm.png
    :align: center

    Attribute table filtered by the filter form

> In the attribute table case, there is also a :guilabel:`Filter features` 

Reviewer: ```suggestion
When selecting / filtering features from the attribute table, there is a :guilabel:`Filter features` 
```

> button that allows filtering features to list. Its use triggers the

Reviewer: What does the "Filter features" button in the table case (view?) look like?  I can't find it in 3.4.9 nor 3.9.

Author: There's not. This is a special feature for form view. Why are you looking for it there?

Reviewer: Sorry. What made me wonder was the phrase "attribute table case".  I can find a "Filter features" button / drop-down in the attribute table "Select/filter features using form" view, where you can define and refine filters. Is this what is meant?

Author: I'd have bet :)
To avoid the description of the tools at the bottom of the attribute table in form view, tools that are not there in normal context (ie, if you are not filtering features), we refer to the "select features by value" dialog description. The main difference between the two functions is that the "select features by value" dialog relies on feature form dialog (and not the attribute table) and DOES NOT have the "feature filter" drop-down menu.
I think the changes should be reviewed with global and html context in mind (it relies on the previous section) and not only as a github diff.

Reviewer: I always try do review with a global context (looking at the preceding sections). I try to help make the text easier to read and understand.

> button that allows filtering features to list. Its use triggers the

Reviewer: ```suggestion
button that allows defining and refining filters. Its use triggers the
```
